app/models/edition.rb

```diff
@@ -7,11 +7,22 @@ class Edition < ActiveRecord::Base
   belongs_to :guide
   belongs_to :user
 
+  has_many :approvals
+
   scope :draft, -> { where(state: 'draft') }
   scope :published, -> { where(state: 'published') }
+  scope :review_requested, -> { where(state: 'review_requested') }
 
   validates_presence_of [:state, :phase, :description, :title, :update_type, :body, :publisher_title, :publisher_href, :user]
-  validates_inclusion_of :state, in: %w(draft published)
+  validates_inclusion_of :state, in: %w(draft published review_requested approved)
+  validate :has_been_approved?
+
+  def has_been_approved?
+    return if !published?
+    if self.approvals.empty?
+      errors.add(:state, "This guide must be approved at least once before publishing")
```

Comment:
I think it would be better to say: "All changes need to be approved by another Content Designer before publishing". it's the changes they're approving, not the guide itself.